vignettes/examples.Rmd

```diff
@@ -0,0 +1,350 @@
+---
+title: "Examples of Usage"
+author:
+- name: "Arash Modaresi Rad"
+    url: https://github.com/arashmodrad
+    affiliation: Lynker
+    affiliation_url: https://lynker.com
+  - name: "Mike Johnson"
+    url: https://github.com/mikejohnson51
+    affiliation: Lynker
+    affiliation_url: https://lynker.com
+output: distill::distill_article
+vignette: >
+  %\VignetteIndexEntry{intro}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+- [Examples](#Examples)
```

Comment:
pkgdown will build a TOC so this can be removed

Reply:
this is neat. sure